driver/run.go

```diff
@@ -25,8 +25,11 @@ func (d *Driver) SetWarningsWriter(w io.Writer) {
 }
 
 func (d *Driver) Write(cid int, arr zbuf.Batch) error {
+	if len(d.writers) == 1 {
```

Comment:
nit: is this check & cid enforcement necessary?

Reply:
This check (and the `cid = 0` below it) serve to maintain the default pre-existing behavior, which is to multiplex all flowgraph leaves into a single output.